solutions/platinum/cses-1706.mdx

```diff
@@ -0,0 +1,130 @@
+---
+id: cses-1706
+source: CSES
+title: School Excursion
+author: Brebenel Mihnea
+---
+
+## Explanation
+
+The solution is based on DSU ([Disjoin Set Union](https://usaco.guide/gold/dsu?lang=cpp))
```

Comment:
```suggestion
The solution is based on DSU ([Disjoint Set Union](/gold/dsu))
```

as mentioned in https://usaco.guide/general/adding-solution?lang=cpp

Reply:
Done